ns/company.go

```diff
@@ -0,0 +1,39 @@
+package ns
+
+import (
+	"encoding/json"
+	"net/http"
+	"os"
+)
+
+// CompanyRequest The structure of the request will be made to the company endpoint.
+type CompanyRequest struct {
+	Credentials *Credentials `json:"credentials"`
+}
+
+// CompanyService operates over company requests.
+type CompanyService service
+
+// Companies returns all companies.
+func (cs *CompanyService) Companies(arq *CompanyRequest) (cl []*CompanyList, err error) {
```

Comment:
I know this was copied/pasted from what I did previously, but the variable here sould be consistent with the type. In this case I'd suggest `crq` instead of `arq`.